awx/ui_next/src/screens/Template/JobTemplateAdd/JobTemplateAdd.jsx

```diff
@@ -36,6 +38,16 @@ function JobTemplateAdd() {
   }
 
   async function submitLabels(templateId, labels = [], orgId) {
+    if (!orgId) {
```

Comment:
I might be misremember the conversation me, you, @marshmalien  and I had, but I thought we were trying to use the JT's inventory organization value fo this rather than making calls.

Reply:
there are times when there is no org, even from a project. For example,  if I go to edit the template, but don't change the project (it's value is just the project id) and don't have an inventory (it's prompt on launch) then we don't have access to an org id and need to fetch 1.